### DISCUSSION TITLE
Firewall open ports on troubleshooting, missing port 8009 and 4443

### DISCUSSION THREAD

**Discussion by DeNB3rt** [mozilla/hubs#5792] [General Questions & Discussion]
Hi all,

In our school environment, every port in-and outbound is blocked.
I followed the guide (https://hubs.mozilla.com/docs/hubs-troubleshooting.html) to open:
-external ports 80 and 443, both via TLS. 
-Ideally, you should also open up ports 51610-65535 on UDP and TCP
-open ports 49152-60999 on UDP and TCP. 
-You will also need to open port 19302 on UDP and TCP.

But this was not enough for us, I found out with wireshark, that **8009** and **4443** was also used.
After opening those two, hubs was running like a charm.

Could you please add this to the troubleshooting page?
Thanks!

**Comment by ajlennon** [comment]
That was helpful - thanks!
